.cirrus.yml

```diff
@@ -123,6 +126,64 @@ task:
       pg-ci-${CIRRUS_TASK_NAME}-${DATE}
 
 
+task:
+  matrix:
+    - name: openbsd-7-0
+      env:
+        PACKERFILE: packer/openbsd.pkr.hcl
+
+    - name: netbsd-9-2
+      env:
+        PACKERFILE: packer/netbsd.pkr.hcl
+
+  compute_engine_instance:
+    image_project: pg-ci-images-dev
+    image: debian-bullseye-nv
+    platform: linux
+    cpu: 4
+    memory: 16G
```

Comment:
Do we really need this much memory? That'll reduce the number of concurrent cirrus tasks we do noticeably.